src/server/graphql/types/Project.js

```diff
@@ -23,11 +29,11 @@ const Project = new GraphQLObjectType({
       type: GraphQLID,
       description: 'The userId that created the project'
     },
-    cursor: {
-      type: GraphQLISO8601Type,
-      description: 'the pagination cursor (createdAt)',
-      resolve({createdAt}) {
-        return createdAt;
+    editors: {
+      type: new GraphQLList(ProjectEditorDetails),
+      description: 'a list of users currently editing the project (fed by a subscription, so queries return null)',
+      resolve: ({editors = []}) => {
```

Comment:
we don't actually store editors in the database because that'd be a lot of work to keep accurate (what if our DB or a client crashes without "leaving" a project?). So, when you query, you'll always get an empty array, but when you subscribe, you'll get editors as they come & go.

Reply:
same problem exists if we kept it in redis. if a client crashes, we don't know that they are no longer editing the card. On disconnect we'd have to run a mutation that looked across every single card that they could possibly be editing. Not impossible, but so much easier to do it this way.